tox.ini

```diff
@@ -26,6 +26,9 @@ ignore =
     D102 # method docstring
     D103 # function docstring
     D107 # init docstring
+    E203 # black
+    E231 # black
+    E501 # black
```

Comment:
I feel like we should add a bit more clear comment why these are inserted (non pep8 compliant)

Reply:
https://github.com/ComPWA/tensorwaves/pull/49/commits/62e5d1e044829be755c55de3ddce81475dde5fdf